bom/pom.xml

```diff
@@ -59,212 +63,216 @@
     </developer>
   </developers>
 
+  <properties>
+    <!-- Keep in sync with ../pom.xml -->
+    <tcnative.version>2.0.43.Final</tcnative.version>
+  </properties>
   <dependencyManagement>
     <dependencies>
       <!-- All release modules -->
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-buffer</artifactId>
-        <version>${project.version}</version>
+        <version>4.1.69.Final-SNAPSHOT</version>
```

Comment:
If `${project.version}` works why not avoid having to make many boilerplate changes for each update? Handled by `maven-release-plugin`?

Reply:
It's done by the release plugin

Reply:
I think it would still be nicer to have them as project.version, especially given they were already swapped to be so in the prior change.

Its nice when the release plugin doesnt need to make unnecessary version changes across an entire file, makes it easier to see the more important detail that really has changed between any particular revisions.

Reply:
I raised #11698 to reinstate the swap to project.version from @Scottmitch earlier changes in https://github.com/netty/netty/commit/d6383bf2478267eb62b1f71807c2a3c34794d1b0